peripherals/flash/flash_wl.yaml

```diff
@@ -156,7 +156,7 @@ FLASH:
         Done: [0, "Options modification completed or idle"]
       _write:
         Start: [1, "Trigger options programming operation"]
-    PNB: [0, 0x63]
+    PNB: [0, 0x7F]
```

Comment:
I think this is an error in the STM32WLE reference manual, the STM32WLE manual only goes from page 0 - 63 (the value should be 0x3F in that case), but in the STM32WL5X reference manual this goes from 0 - 127.

As far as I can tell the STM32WLE is capable of programming its whole 256K, same as the STM32WL5X.